bluetooth/src/iosMain/kotlin/device/DefaultDeviceConnectionManager.kt

```diff
@@ -135,7 +135,7 @@ internal actual class DefaultDeviceConnectionManager(
         else -> DeviceConnectionManager.State.DISCONNECTED
     }
 
-    actual override fun connect() {
+    actual override suspend fun connect() {
```

Comment:
If I look at this right, this introduces a platform inconsistency.
On Android we wait until connection completes
On iOS, we just say connect and expect the State machine to track it later on.